tests/unit/h_testkit/test_it.py

```diff
@@ -0,0 +1,2 @@
+def test_it():
+    assert True
```

Comment:
There has to be at least one test otherwise pytest fails. But I don't think it's worth writing any actual tests for this library yet.